tests/test-refs.sh

```diff
@@ -65,4 +65,25 @@ ${CMD_PREFIX} ostree refs --repo=repo | wc -l > refscount.delete3
 assert_file_has_content refscount.delete3 "^3$"
 assert_not_file_has_content reflist '^test-1$'
 
+#Add a few more commits, to test --create
+${CMD_PREFIX} ostree --repo=repo commit --branch=ctest -m ctest -s ctest tree
+${CMD_PREFIX} ostree --repo=repo commit --branch=foo/ctest -m ctest -s ctest tree
+
+${CMD_PREFIX} ostree --repo=repo refs | wc -l > refscount
+assert_file_has_content refscount "^5$"
+
+${CMD_PREFIX} ostree --repo=repo refs --create=ctest-new 2>/dev/null && (echo 1>&2 "refs --create unexpectedly succeeded without specifying an existing ref!"; exit 1)
```

Comment:
This is OK, but elsewhere we use a lot the style:

```
if command-that-should-fail; then
  assert_not_reached "command-that-should-fail unexpectedly succeeded!"
fi
```

it's a bit more lines, but easier to understand I think.